### DISCUSSION TITLE
Does groupBy work with Dates?

### DISCUSSION THREAD

**Discussion by hughess** [pbeshai/tidy#46] [Q&A] [answered]
I'm summarizing a column of values and grouping it by a date column (which is stored as a JS Date object). Should this work? At the moment it appears to be treating instances of the same date as separate groups. I worked around this for now using Date.parse so it is comparing numbers instead

**Comment by pbeshai** [comment, marked as answer]
Ah this is a bug, thank you. I will post a fix momentarily. 

This is fixed in v2.4.3

**Reply by hughess** [reply]
Great, thanks - love this package!
